src/pages/contact-us.js

```diff
@@ -0,0 +1,19 @@
+import { Box, Flex, Text } from "@chakra-ui/react";
+import { Layout } from "../components/layout";
+import ContactForm from "../components/contact-form";
```

Comment:
Use a named export for exporting the component. `export function ContactForm()` instead of `export default`. We only use export default when we have to, which is very rare.

Reply:
Pokusao sam da uklonim default ispred funkcije ali mi je izbacivao compailer error. So i just imported ContactForm as Form